nextflow_schema.json

```diff
@@ -29,6 +29,11 @@
                     "description": "The output directory where the results will be saved. You have to use absolute paths to storage on Cloud infrastructure.",
                     "fa_icon": "fas fa-folder-open"
                 },
+                "refgenome": {
+                    "type": "string",
+                    "description": "The file path to the FASTA-format reference genome to use.",
```

Comment:
This should have `format` of `file-path` as described in https://nextflow-io.github.io/nf-validation/nextflow_schema/nextflow_schema_specification/#format

Reply:
Added this and file extension checking in e75f97b18ff93a4adf4da0f7007190b722e24f7f